### ISSUE TITLE
[Feature] Set "Initializing Skin" Screen to run until Ratings are loaded

### ISSUE THREAD

**Issue by cboRD181** [jurialmunkey/skin.arctic.horizon.2#695]
### Skin section

Other

### Feature description

An option to set the beautiful "Initializing Skin" screen ([Custom_1198_Startup.xml](https://github.com/jurialmunkey/skin.arctic.horizon.2/blob/main/1080i/Custom_1198_Startup.xml)) to run until the home widgets are completely loaded -- meaning, ratings have been fetched and are displayed, and (ideally) addon powered widgets are displaying properly, e.g. episodes instead of a loading circle. 

### Why is this feature needed? Please justify and explain.

At present the "Initialising Skin" screen only shows for a few seconds and gives the impression that the content is loaded, but impactful parts of the experience are still warming up.  Waiting until the ratings and other widget content has loaded would give both a more intuitive and enjoyable experience, imo. 

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Not possible to wait for the widgets to load because Kodi will only start loading them once the home screen is open (i.e. the skin needs to close the Startup window first).

However, I've added an option to wait for the plugin system to initialise. That will wait on the Startup screen until Kodi has initialised python and TMDbHelper can connect to the internet and authorise Trakt etc. -- there will still some delay on the home screen while widgets load but it should mask a lot of the wait time.

`Skin Settings > Behaviour > Wait for plugins to initialise on startup`

**Comment by cboRD181** [comment]
Awesome! I'll check it out now, thanks

**Comment by jurialmunkey** [comment]
> Awesome! I'll check it out now, thanks

I haven't made a release with it yet, so you'll need to get the zip from the main branch rather than the releases section (or you can wait a bit and I'll push a release soon - just want to add a few other things first).

**Comment by xyzfre** [comment]
> Not possible to wait for the widgets to load completely because Kodi will only start loading them once the home screen is open (i.e. the skin needs to close the Startup window first for widgets to start loading).
> 
> 
> 

> 
> 
> 
> ^^^ With that enabled, the skin will wait until Kodi has initialised python and TMDbHelper can connect to the internet and authorise Trakt etc. before opening the home screen. There will still be some delay while the widgets load but it should mask a lot of the "dead" time spent waiting for the plugin system to startup.

Thank you @jurialmunkey for this new setting realize yesterday ,, definitely makes a big difference on start up 💯💯 ,, I'm on Nexus RC2 Windows 11 Pro everything is working amazing GREAT WORK❣️
